custom_components/ecowitt/__init__.py

```diff
@@ -274,6 +283,33 @@
     TYPE_DEWPOINTC: ("Dewpoint", TEMP_CELSIUS,
                      TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
                      "mdi:thermometer", 0),
+    TYPE_DEWPOINTINC: ("Indoor Dewpoint", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
+    TYPE_DEWPOINT1C: ("Dewpoint 1", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
+    TYPE_DEWPOINT1C: ("Dewpoint 2", TEMP_CELSIUS,
+                     TYPE_SENSOR, DEVICE_CLASS_TEMPERATURE,
+                     "mdi:thermometer", 0),
```

Comment:
Oops, you repeated 1C over and over here..  :)

Reply:
whoops, will fix and re-push tonight.

Reply:
Re-pushed the commit with changes